drydock/patches/kustomization

```diff
@@ -0,0 +1,55 @@
+patches:
+{% if DRYDOCK_NEWRELIC -%}
+- target:
+      kind: Deployment
+      name: lms
+  path: plugins/drydock/k8s/newrelic.yml
+{% endif -%}
+- target:
+    kind: Job
+    labelSelector: app.kubernetes.io/component=job
+  patch: |-
+    apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: does-not-matter
```

Comment:
Maybe a Json6902 would be better in this case as to avoid those placeholders in name? I think we can inline it


```yaml
  patch: |-
    - op: add
      path: /metadata/annotations
      value:
        argocd.argoproj.io/hook: Skip
        argocd.argoproj.io/hook-delete-policy: HookFailed
    - op: add
      path: /spec
      value:
        ttlSecondsAfterFinished: 100
```